src/components/NavigationAppBar.js

```diff
@@ -119,26 +134,60 @@ class NavigationAppBar extends React.Component {
-              <IconButton color="inherit" onClick={() => onChangePage(LANDING)}>
-                <ArrowForwardIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(INFO)}>
-                <InfoIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(IMPORT)}>
-                <InputIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(CATEGORIZE)}>
-                <CategoryIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(VISUALIZE)}>
-                <ShowChartIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(EXPORT)}>
-                <PublishIcon />
-              </IconButton>
+              <Tooltip title={<FormattedMessage id="navigation.start" />}>
+                <IconButton
+                  color="inherit"
+                  onClick={() => onChangePage(LANDING)}
+                  className={activePage === LANDING ? classes.active : ''}
+                >
+                  <ArrowForwardIcon />
+                </IconButton>
+              </Tooltip>
+              <Tooltip title={<FormattedMessage id="navigation.info" />}>
+                <IconButton
+                  color="inherit"
+                  onClick={() => onChangePage(INFO)}
+                  className={activePage === INFO ? classes.active : ''}
+                >
+                  <InfoIcon />
+                </IconButton>
+              </Tooltip>
+              <Tooltip title={<FormattedMessage id="navigation.import" />}>
+                <IconButton
+                  color="inherit"
+                  onClick={() => onChangePage(IMPORT)}
+                  className={activePage === IMPORT ? classes.active : ''}
+                >
+                  <InputIcon />
+                </IconButton>
+              </Tooltip>
+              <Tooltip title={<FormattedMessage id="navigation.categorization" />}>
+                <IconButton
+                  color="inherit"
+                  onClick={() => onChangePage(CATEGORIZE)}
+                  className={activePage === CATEGORIZE ? classes.active : ''}
+                >
+                  <CategoryIcon />
+                </IconButton>
+              </Tooltip>
+              <Tooltip title={<FormattedMessage id="navigation.visualization" />}>
+                <IconButton
+                  color="inherit"
+                  onClick={() => onChangePage(VISUALIZE)}
+                  className={activePage === VISUALIZE ? classes.active : ''}
+                >
+                  <ShowChartIcon />
+                </IconButton>
+              </Tooltip>
+              <Tooltip title={<FormattedMessage id="navigation.export" />}>
+                <IconButton
+                  color="inherit"
```

Comment:
I think having the UI like this would be more pleasant to my eyes, but it's up to you:

<img width="1395" alt="screenshot 2018-11-08 at 11 49 43" src="https://user-images.githubusercontent.com/18005239/48190879-741edf00-e34c-11e8-8fd1-1ade5bb5efdf.png">

So the selected item would get darker. I managed to get it like that by setting `color="default"`